Neos.Neos/Resources/Private/Templates/Login/Index.html

```diff
@@ -2,6 +2,7 @@
 <f:layout name="Default" />
 
 <f:section name="head">
+	<meta name="robots" content="noindex">
```

Comment:
```suggestion
	<meta name="robots" content="noindex"/>
```
Self closing tag, not sure if it matters but XHTML for the win ;)

Reply:
It's valid for void elements in HTML5, but the optional `/` is also allowed (except for the massive increase in file size (jk jk) 😝) When looking at our other templates in `Neos.Neos` we usually have `<meta ...  />` so to keep things consistent this is a good idea.

Reply:
I started looking for those slashes again, as moving code from fluid to AFX will break without them.

Reply:
Ah yea, guess the AFX lexer will always need them!